web/static/js/components/stage_help.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import ReactDOM from "react-dom"
+
+import { connect } from "react-redux"
+import { bindActionCreators } from "redux"
+import { actions } from "../redux/retro"
+
+import * as AppPropTypes from "../prop_types"
+import STAGES from "../configs/stages"
+
+export const StageHelp = props => {
+  const handleClick = () => {
+    const { retro, actions: { showStageHelp } } = props
+    showStageHelp(retro)
+  }
+
+  const showIcon = () => {
+    const { retro: { stage } } = props
+    const validHelpStages = [
+      STAGES.PRIME_DIRECTIVE,
```

Comment:
hmmm... this datastructure is implicitly coupled to the stage config object, as the only stage omitted here is the one state where `help` is null. so maybe we could just key into that object and `showIcon` if and only if `!help`?

Reply:
Cool - good idea.

Reply:
The internet knows all.